iOSDFULibrary/Classes/Implementation/SecureDFU/Services/SecureDFUService.swift

```diff
@@ -461,14 +461,14 @@ import CoreBluetooth
      
      - parameter report: method called when an error occurred
      */
-    func jumpToBootloaderMode(onError report: @escaping ErrorCallback) {
+    func jumpToBootloaderMode(dontSetAdvertisingName: Bool, onError report: @escaping ErrorCallback) {
         if !aborted {
             func enterBootloader() {
                 self.buttonlessDfuCharacteristic!.send(ButtonlessDFURequest.enterBootloader, onSuccess: nil, onError: report)
             }
             
             // If the characteristic may support changing bootloader's name, try it
-            if buttonlessDfuCharacteristic!.maySupportSettingName {
+            if dontSetAdvertisingName == false && buttonlessDfuCharacteristic!.maySupportSettingName {
```

Comment:
This should be renamed to match the property if it's set
, something like `alternativeAdvertisingFeatureEnabled` would make this a bit more readable ?